metadatastore/commands.py

```diff
@@ -495,7 +493,25 @@ def find_event(run_start):
         Set of events encapsulated within a RunStart's scope.
         metadatastore.document.Document
     """
-    descriptors = EventDescriptor.objects(run_start=run_start.id)
+    return find_event_by_runstart_id(run_start.id)
+
+
+@_ensure_connection
+def find_event_by_runstart_id(run_start_id):
+    """Returns a set of events given a RunStart object
+
+    Parameters
+    ---------
+    run_start_id: mongoengine.ObjectID
```

Comment:
This works using the method Tom and I have agreed upon earlier:

https://github.com/arkilic/metadatastore/commit/b70253e53318369a3702e600af197be16056be07